docs/license.md

```diff
@@ -0,0 +1,636 @@
+# GNU GENERAL PUBLIC LICENSE
+Version 3, 29 June 2007
```

Comment:
We cannot have a license file in here. When we move the distro to Drupal.org the packager will add a license file. That license will conflict with this one then.